vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -0,0 +1,53 @@
+| Entando Content Attribute | The basic data that define a content type |
+| Entando Content Template | The style or layout of a content type where a single content type can have multiple content models defining different ways to render the same content |
+| Entando Digital Asset | An image, document, or other media file that is uniquely identifiable and stored digitally in a format supported by the Entando Platform |
+| Entando Hub |  A repository (local, remote, public, or private) containing components built with the Entando Platform; A single Entando App Builder can connect to 1 or more Entando Hubs |
+| Entando Identity Management | Entando's Keycloak- and token-based user management and authentication system |
+| Entando Kubernetes integration service (k8s-service) | A function of the Entando Cluster infrastructure custom resource that provides an abstraction layer between Entando microservices and the APIs exposed by Kubernetes |
+| Entando Local Hub | The local component repository in an Entando App Builder used to store and retrieve shareable components for reuse within an Entando Application |
+| Entando Operator | Provides installation and application lifecycle automation for Entando Applications, microservices and required infrastructure services, e.g. databases and Keycloak |
+| Entando Platform | The leading open source application composition platform for development with a Kubernetes architecture |
+| Entando Platform Capability (EPC) | A packaged capability that adds functionality to the platform and/or additional UX controls to the App Builder |
+| Entando Plugin | A microservice that exposes APIs reusable by one or more Entando Applications |
+| Entando UX Fragment | An HTML block containing Freemarker tags that allow content to be rendered dynamically according to context |
+| Entando Web Content Management System (WCMS) | Entando's lightweight content and digital asset management system that supports headless operations |
+| Entando Widget | A UI element that can be dragged and dropped onto a page, e.g. a snippet of HTML code added from the browser or a micro frontend, which is a specialized widget |
+
+
+## General Technical Terms
+
+| Term |  Description
+|:--|:--
+| application composition platform (ACP) | Any development platform that supports the cataloging and management of composable and packaged components, where new components can be added through custom development or imported from existing assets; it governs the life cycles of both the components and the applications built from their modular assembly and deployment |
+| backend for frontend (BFF) | A type of microservice dedicated to a frontend that may also act as a facade to other enterprise microservices |
+| component | An application building block such as a page template, content template, UX Fragment, widget, micro frontend, microservice, etc. |
+| component collection | A packaged set of single components that are in some way functionally unrelated |
```

Comment:
Remove "that are in some way functionally unrelated". There's no requirement that the components are or are not related.

Reply:
do you know who originally wrote this or can we just go ahead and change it without discussion

<img width="894" alt="Screen Shot 2022-11-18 at 6 18 25 PM" src="https://user-images.githubusercontent.com/93157274/202829733-effd5eee-c843-43ab-938b-7989ed63a191.png">